src/http/server.rs

```diff
@@ -21,15 +21,18 @@ fn graphiql() -> Result<Response<Body>> {
   )))
 }
 
-async fn graphql_request(req: Request<Body>, server_ctx: &ServerContext) -> Result<Response<Body>> {
+async fn graphql_request(
+  req: Request<Body>,
+  server_ctx: &ServerContext,
+  executor: Arc<dyn RequestExecutor + Send + Sync>,
```

Comment:
```suggestion
  executor: Arc<impl RequestExecutor + Send + Sync>,
```

This should improve performance.